File: Backend/Movieverse.Contracts/DataTransferObjects/Person/PersonDto.cs
using Movieverse.Domain.ValueObjects;

namespace Movieverse.Contracts.DataTransferObjects.Person;

public sealed class PersonDto
{
	public Information Information { get; set; } = null!;
	public LifeHistory LifeHistory { get; set; } = null!;
	public string? Biography { get; set; }
	public string? FunFacts { get; set; }
	public Guid? PictureId { get; set; }
	public List<Guid> ContentIds { get; set; } = new();
}

File: Frontend/movieverse/src/core/dtos/person/personDto.ts
import { InformationDto } from "../user/userDto"

export type PersonDto = {
  information: InformationDto;
  lifeHistory: LifeHistoryDto;
  biography?: string;
  funFacts?: string;
  pictureId?: string;
  contentsIds: string[];
}

export type LifeHistoryDto = {
  birthPlace?: string;
  birthDate?: string;
  deathPlace?: string;
  deathDate?: string;
  careerStart?: string;
  careerEnd?: string;
}